src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml

```diff
@@ -392,15 +392,20 @@
           p:emailService-ref="emailService"
           p:scheduledThreadPool-ref="rice.ksb.scheduledThreadPool"
           p:exceptionMessageJob-ref="exceptionMessageJob"
+          p:delayedAsyncCallJob-ref="delayedAsyncCallJob"
           p:useQuartzScheduling="${use.quartz.scheduling}"/>
 
     <bean id="messageServiceExecutorJobListener"
           class="org.kuali.kfs.ksb.messaging.quartz.MessageServiceExecutorJobListener"/>
 
     <!--
-        This bean is only meant for KSB-message-related processing, so we exclude it
+        These beans are only meant for KSB-message-related processing, so we exclude it
```

Comment:
Now that the comment references multiple beans, I would suggest one other quick change: "...exclude it" --> "...exclude them"

Reply:
I will update